plugins/snowflake/dbt/include/snowflake/macros/materializations/table.sql

```diff
@@ -35,6 +35,8 @@
   {{ adapter.drop_relation(intermediate_relation) }}
 
   -- setup: if the target relation already exists, truncate or drop it (if it's a view)
+  {# TODO: Would like to check this. New materialsiation makes these tests a bit moot. We should
+  be able to deprecate non-destructive flag all together here. #}
   {% if non_destructive_mode -%}
```

Comment:
I'd like to raise attention to this part. I left the remanants of the use of non destructive but I have a feeling we can just do away with that flag being used all together and do special handling for views if they create an issue. But I thought I'd check with one of you guys